data/brands/24s.yaml

```diff
@@ -0,0 +1,22 @@
+name: 24S
+status: avoid
+description: "**24S** is owned by **LVMH**.\n\nBernard Arnault, the chairman and **CEO**\
```

Comment:
i wonder what it is about this description that makes it formatted like `description: "some long sentence"`.. but other descriptions are formatted with `description: |-`

i prefer the second format - if you can figure out how to script it that would be awesome.. if not, we can create a follow up task to standardise the format of this text

Reply:
@THM222 I have fixed this in my latest commit - it was due to some characters in the strings that YAML won't allow in block strings (e.g. spaces before new lines, certain unicode characters, trailing whitespace).